driver/conv_driver.hpp

```diff
@@ -1564,7 +1564,7 @@ void ConvDriver<Tgpu, Tref>::PrintForwardTime(const float kernel_total_time,
         std::tie(out_n, out_c, out_h, out_w) =
             miopen::tien<4>(miopen::deref(outputTensor).GetLengths());
 
-        size_t flopCnt = 2L * in_n * in_c * wei_h * wei_w * out_c * out_h * out_w / group_count;
+        size_t flopCnt = 2LL * in_n * in_c * wei_h * wei_w * out_c * out_h * out_w / group_count;
```

Comment:
Is it better to use `ULL`? We've already cast to `size_t` which is unsigned.
The same applies to the rest `LL` changes.

But for the other places `static_cast<size_t>` used for literals with `UL`.

Reply:
This review comment is duplicated at https://github.com/ROCmSoftwarePlatform/MIOpen/pull/2323#discussion_r1293542099. I recommend marking this one "resolved".

Reply:
Almost. But for other literals `static_cast` to explicit type has been used.
There are two problems - `ULL` vs `LL` and `static_cast` vs `LL`. The comment covers only the first part.

Reply:
I guess you've meant the last comment - it covers the both cases.